components/ai_chat/resources/page/components/edit_input/index.tsx

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+
+import Button from '@brave/leo/react/button'
+import Icon from '@brave/leo/react/icon'
+
+import styles from './style.module.scss'
+
+interface Props {
+  text: string
+  onSubmit: (text: string) => void
+  onCancel: () => void
+}
+
+function EditInput(props: Props) {
```

Comment:
![alt text](image.png) - when there's multiline text should the action buttons are rendered like this?

Reply:
<img width="666" alt="image" src="https://github.com/brave/brave-core/assets/8665427/5a97d7ec-a1ed-4f42-a79d-ab3876b80964">

Reply:
<img width="646" alt="Screenshot 2024-07-10 at 5 27 42 PM" src="https://github.com/brave/brave-core/assets/4730197/90fa0a69-2a4a-4daf-a47e-e6d4b7b42093">
Updated after updated design.